README.md

```diff
@@ -1,10 +1,22 @@
 # sul-solr-configs
 
-This repository manages the Solr configuration files in our solrcloud.
+This repository manages the Solr configuration files in our SolrCloud.
 
 ## Required configuration
 
-To run a solr collection in the cloud, you must include some required configuration. For Solr 4.x, the [required solr configuration](https://wiki.apache.org/solr/SolrCloud#Required_Config) is documented on the Solr wiki.
+To run a Solr collection in our SolrCloud, you must include some required configuration. For Solr 4.x, the [required solr configuration](https://wiki.apache.org/solr/SolrCloud#Required_Config) is documented on the Solr wiki.
```

Comment:
this link goes to solr 6.6 docs (for me, at least). should we change the text that reads "4.x"? are we even running v6? I thought we were on like v8

Reply:
we are Solr 8.11;   this comment was old and I decided to leave the opportunity for someone else to update it in a future PR.  :-P

Reply:
I guess it at least tells you that it's documentation for an older version 🤷🏻